src/ast/passes/resolve_imports.cpp

```diff
@@ -96,44 +49,26 @@ bool ResolveImports::checkPerms(const std::filesystem::path &path)
   }
 }
 
-Result<OK> ResolveImports::importScript(Node &node,
-                                        const std::string &name,
-                                        const std::filesystem::path &path)
+static Result<OK> importScript([[maybe_unused]] Node &node,
+                               Imports &imports,
+                               const std::string &name,
+                               const std::string &&contents,
+                               const std::vector<std::filesystem::path> &paths,
+                               std::map<std::string, ASTContext> &where)
```

Comment:
Don't love this variable name... why not call it `scripts` ?

Reply:
I'll just push back gently -- this is the map where the contents are placed. I could call it `out`? I would just like to keep it consistent with the other names for the other cases, and `scripts` is not suitable for those ones.

Reply:
What about `cache` or `contents` ?